apple_server.js

```diff
@@ -67,8 +67,9 @@ const verifyAndParseIdentityToken = (query, idToken, isNative = false) =>
  * @param query
  * @param {*} tokens tokens and data from apple
  * @param isNative
+ * @param isBeingCalledFromLoginHandler
  */
-const getServiceDataFromTokens = (query, tokens, isNative = false) => {
+const getServiceDataFromTokens = (query, tokens, isNative = false, isBeingCalledFromLoginHandler = false) => {
```

Comment:
it would be better to change this to an options object. So we don't need to rely on order.